packages/calcite-components/src/components/panel/panel.scss

```diff
@@ -2,11 +2,30 @@
  * CSS Custom Properties
  *
  * These properties can be overridden using the component's tag as selector.
- *
- * @prop --calcite-panel-content-space: Specifies the padding of the component's content.
- * @prop --calcite-panel-footer-padding: Specifies the padding of the component's footer.
- * @prop --calcite-panel-header-border-block-end: Specifies the component header's block end border.
+
+ * @prop --calcite-panel-corner-radius: Specifies the corner radius of the component.
+ * @prop --calcite-panel-heading-text-color: Specifies the text color of the component's `heading`.
+ * @prop --calcite-panel-description-text-color: Specifies the text color of the component's `description`.
+ * @prop --calcite-panel-border-color: Specifies the border color of the component.
+
  * @prop --calcite-panel-background-color: Specifies the background color of the component.
+ * @prop --calcite-panel-header-background-color: Specifies the background color of the component's footer.
+ * @prop --calcite-panel-footer-background-color: Specifies the background color of the component's footer.
+
+ * @prop --calcite-panel-space: Specifies the padding of the component's `"unnamed (default)"` slot.
+ * @prop --calcite-panel-header-content-space: Specifies the padding of the `"header-content"` slot.
```

Comment:
this doesn't appear to do anything. Remove?

Reply:
It's used here: https://github.com/Esri/calcite-design-system/pull/10822/files#diff-0b45120ffa61191421f1699207bb91da11d979e0a76a1b28d50353a3b17394c2R228
 
![Screenshot 2024-11-21 at 10 48 49 AM](https://github.com/user-attachments/assets/a138a635-5175-4dc1-aa86-7190e499b51f)

There's a separate example in the local demo and theme test for this "header-content-space" use case.

Reply:
missed that thanks!